File: Source/Rio.Models/DataTransferObjects/WaterTransfer/WaterTransferRegistrationSimpleDto.cs
using Rio.Models.DataTransferObjects.Account;
using System;

namespace Rio.Models.DataTransferObjects.WaterTransfer
{
    public class WaterTransferRegistrationSimpleDto
    {
        public int WaterTransferRegistrationID { get; set;}
        public int WaterTransferTypeID { get; set; }
        public AccountDto Account { get; set; }
        public int WaterTransferRegistrationStatusID { get; set; }
        public DateTime StatusDate { get; set; }
        public bool IsRegistered { get; set; }
        public bool IsCanceled { get; set; }
        public bool IsPending { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/water-transfer-registration-simple-dto.ts
import { AccountDto } from './account/account-dto';

export class WaterTransferRegistrationSimpleDto {
    WaterTransferTypeID: number;
    Account: AccountDto;
    WaterTransferRegistrationStatusID: number;
    StatusDate: Date;
    IsRegistered: boolean;
    IsCanceled: boolean;
    IsPending: boolean;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

